go.mod

```diff
@@ -52,12 +50,9 @@ require (
 	google.golang.org/appengine v1.4.0 // indirect
 	google.golang.org/genproto v0.0.0-20190201180003-4b09977fb922 // indirect
 	google.golang.org/grpc v1.19.1
-	gopkg.in/bblfsh/client-go.v3 v3.2.1
-	gopkg.in/bblfsh/sdk.v1 v1.16.1 // indirect
```

Comment:
this is interesting. because `client-go`  still depends on sdk.v1 and other removed stuff. But they aren't displayed as indirect anymore 🤔 
(just wondering)

Reply:
@smacker the reason because it happened can be found at [go/Modules#why-does-go-mod-tidy-record-indirect-dependencies](https://github.com/golang/go/wiki/Modules#why-does-go-mod-tidy-record-indirect-and-test-dependencies-in-my-gomod)
>_If a dependency of your module_ (`client-go.v3`) _does not itself have a `go.mod`_ (`v3` had not)_, then the missing transitive dependencies_ (`sdk.v1`) _will be added to your module's requirements, along with an `// indirect` comment._

[`client-go.v3`](https://github.com/bblfsh/client-go/tree/v3.2.1) didn't have a `go.mod`, so its dependencies were added into our `go.mod`.
Since we migrated to [`go-client/v4`](https://github.com/bblfsh/go-client/tree/v4.0.0) and it does have a proper `go.mod`, then its dependencies are not added into ours.